iis-common/src/test/java/eu/dnetlib/iis/common/spark/TestWithSharedSparkContext.java

```diff
@@ -0,0 +1,50 @@
+package eu.dnetlib.iis.common.spark;
+
+import eu.dnetlib.iis.common.SlowTest;
+import org.apache.spark.SparkConf;
+import org.apache.spark.SparkContext;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.util.Objects;
+
+/**
+ * Support for tests using {@link org.apache.spark.SparkContext} - extends this class to access SparkContext in local mode.
+ */
+@SlowTest
+public class TestWithSharedSparkContext {
+    private static transient SparkContext _sc;
+    private static transient JavaSparkContext _jsc;
+    protected boolean initialized = false;
+
+    protected SparkContext sc() {
+        return _sc;
+    }
```

Comment:
Do we really need to have `SparkContext` exposed outside? I didn't find any reference in the changed code. Unless it is exposed for a potential future use...

Reply:
Again, a blind copy from source project, removing the method.